sources/packages/api/src/esdc-integration/e-cert-integration/e-cert-full-time-integration/e-cert-full-time-integration.service.ts

```diff
@@ -175,4 +176,27 @@ export class ECertFullTimeIntegrationService extends ECertIntegrationService {
       OfferingIntensity.fullTime,
     );
   }
+  /**
+   * Populate Full-Time awards from the disbursementValues
+   * @param disbursementValues
+   * @returns awards
+   */
+  populateAwards(disbursementValues: DisbursementValue[]): Award[] {
```

Comment:
since it's only for FT  can you add `full-time` to the method name?

Reply:
its an overwritten method, we cannot rename it.